### PR TITLE
LRQA-84215

### PR DESCRIPTION
**Pull request by Kvale1733**
https://liferay.atlassian.net/browse/LRQA-84215

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro [hunk 1]
```diff
@@ -126,8 +126,13 @@ definition {
 			var portalURL = JSONCompany.getDefaultPortalURL();
 		}
 
+		if (isSet(specificURL)) {
+			var domain = ${specificURL};
+		}
+
 		if (!(isSet(creatorEmailAddress))) {
 			var creatorEmailAddress = JSONUtil2.getDefaultUserEmail();
+			var creatorEmailAddress = JSONUtil2.getDefaultUserEmail(domain = ${domain});
```
[the remark this answers is not in the document]
Comment (Kvale1733) [reply portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro:135]: Oh yeat that shouldn't happen. I'll fix it. It should just be the second one.

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by vicnate5** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/144684

**Comment by vicnate5** [comment]
ci:close

**Pull request closed without merging**